net-analyzer/openvas/openvas-9.0.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1
+
+DESCRIPTION="A remote security scanner"
+HOMEPAGE="http://www.openvas.org/"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="+extras +cli +gsa +ospd ldap radius"
+
+DEPEND="
```

Comment:
This ebuild doesn't seem to compile anything, so this DEPEND is definitely wrong. Also why should this meta package pull all that stuff in extras? The ebuilds themself should pull all those.

Reply:
Ok.This is the biggest one i think.

This workflow effects all ebuilds and it is a bit tricky to handle this.
Let me explain and please correct me if i do something wrong.

This is main OpenVAS ebuild (openvas-9.0.0).So i suppose ( and old maintainer too:) ) if someone needs this package will emerge this one first.This one calls other OpenVAS components for building (ordered | libraries-->scanner-->manager-->security-agent-->ospd-->gvm-tools).

So DEPEND is setted to OpenVAS components.All openvas components have seperate ebuilds because of separate dependencies but same for doc,html support ('extras' -use flag)

OpenVAS Components;

> OpenVAS-Libraries
> OpenVAS-Scanner
> OpneVAS-Manager
> Greenbone-Security-Agent 
> Gvm-Tools
> ospd

OpenVAS-Libraries is the key ebuild which other components depend this one build time.So if someone try to emerge OpenVAS components directly e.g. manager, scanner, security agent, ebuilds will call OpenVAS-Libraries.So all direct emerge attempts will point to OpenVAS-Libraries for successfull build.

Here is the GAP:
----------------------
If someone emerge OpenVAS-Libraries directly instead of Openvas main, whats then? Main OpenVAS ebuild is meaningless here.

Then he/she has to continue emerging all components manually one by one and ordered
-->scanner-->manager-->security-agent e.g. to complete setup which is not logical.

Fixing it? How to?
If i make openvas-libraries-9.0.3 depended to net-analyzer/openvas-9.0.0 for pointing all direct component emerges to main package.This is a bit funny loop while removing package:)

openvas-9.0.0.ebuild
```
DEPEND="
        >=net-analyzer/openvas-libraries-9.0.3
"
```

openvas-libraries-9.0.3.ebuild
```
DEPEND="
        >=net-analyzer/openvas-9.0.0
"
```

Reply:
These circular dependencies can be fixed with PDEPEND.

Check the devmanual and examples from Gentoo tree how it works. If you can't find a solution using that, let me know and let's think of something else.